File: bridge/golang/pipe.go
package main

import (
	"bufio"
	"encoding/json"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	for {
		line, _ := reader.ReadString('\n')
		requestJson := line
		var request interface{}
		json.Unmarshal([]byte(requestJson), &request)
		response := {{ .Method }}(request)
		responseJson, _ := json.Marshal(response)
		fmt.Fprintln(os.Stdout, string(responseJson))
	}
}

File: bridge/javascript/pipe.js
var readline = require('readline');
var op = require('./{{ .OpFile }}');

var rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false
});

rl.on('line', function(line){
    var requestJson = line;
    var responseJson = JSON.stringify(op.{{ .Method }}(JSON.parse(requestJson)));
    console.log(responseJson);
});